### PR TITLE
Condense primary nav and add icon to expanded state 

### PR DESCRIPTION
**Pull request by tiffwong**
Basically, the primary nav will always have the icon visible

### CODE REVIEW

File: packages/manager/src/components/SideMenu.tsx [hunk 1]
```diff
@@ -101,12 +99,20 @@ class SideMenu extends React.Component<CombinedProps> {
               keepMounted: true // Better open performance on mobile.
             }}
           >
-            <PrimaryNavComponent
-              closeMenu={closeMenu}
-              toggleTheme={toggleTheme}
-              toggleSpacing={toggleSpacing}
-              isCollapsed={false}
-            />
+            {flags.cmr ? (
```
Comment (tiffwong) [inline packages/manager/src/components/SideMenu.tsx:102]: Had to do this so linter would stop complaining about the missing `toggleTheme` prop in the new nav

File: packages/manager/src/components/PrimaryNav/PrimaryNav_CMR.styles.ts [hunk 2]
```diff
@@ -77,24 +67,19 @@ const useStyles = makeStyles((theme: Theme) => ({
       }
     },
     '& .icon': {
+      color: theme.cmrIconColors.iGrey,
```
Comment (tiffwong) [inline packages/manager/src/components/PrimaryNav/PrimaryNav_CMR.styles.ts:70]: Color is still tbd

File: packages/manager/src/components/SideMenu.tsx (hunk 1 above)
Comment (Jskobos) [reply packages/manager/src/components/SideMenu.tsx:102]: Easier to burn the non-cmr stuff to the ground, since we're cleared to release

**Review by Jskobos — APPROVED** [review]
lgtm. I'd suggest cleaning out the non-cmr code rather than try to accommodate both sets of props.

**Review by johnwcallahan — APPROVED** [review]

**Pull request merged**